crates/telio-traversal/src/endpoint_providers/upnp.rs

```diff
@@ -36,7 +36,7 @@ const GET_INTERFACE_TIMEOUT_S: Duration = Duration::from_secs(2);
 
 const EPHEMERAL_PORT_RANGE: std::ops::RangeInclusive<u16> = 49152..=65535;
 
-const DEFAULT_LEASE_DURATION_S: u32 = 600;
+const LEASE_RENEWAL_LIMIT_S: u32 = 600;
```

Comment:
What does this const controll, we have what we set as wanted, but i don't entirely get there this comes from.
At what poin we should update lease ?

Reply:
We have `FeaturesUPnP::lease_duration_s` which sets the actual lease duration for the UPnP port mappings. This `LEASE_RENEWAL_LIMIT_S` is the value which when the lease duration falls below it, we should renew the mappings by sending a request to the UPnP device. 

E.g: We set the mappings lease duration to 3600s, when we check the lease duration left - every upnp_interval tick time - and there's less than 600s left for them to expire, we simple renew the lease for another `FeaturesUPnP::lease_duration_s` which is 3600s.

Reply:
Thinking this better, maybe this const can be correlated with the `FeaturesUPnP::lease_duration_s`, something like 1/5 of `FeaturesUPnP::lease_duration_s`

Reply:
Yeh, that was what i was thinking also, maybe min(1/10, 1)

Reply:
It has to be higher than the tick time (`upnp_interval`) though, otherwise it will get expired. I'll try to come up with something.